script/fetch-issues.js

```diff
@@ -223,24 +379,47 @@ function generateRepoInfo(owner, names, issues) {
   return result;
 }
 
+function filterRecent(issues) {
+  // only keep issues created/updated in the past month
+  const today = new Date();
+  const dayLastMonth = today.setMonth(today.getMonth() - 1);
+  return issues.filter(issue => getTime(dayLastMonth) < issue.updatedAtMs);
```

Comment:
`dayLastMonth` never changes, I think it's better to call `getTime` in the `dayLastMonth` variable itself.